app/views/records/edit_fields/_creator.html.erb

```diff
@@ -34,7 +34,7 @@
 		<% end %>
 
 		<% field_options = { field_slug: :creator_isni, required: false, field_group: :creator, template: template, hash: hash } %>
-		<%= render "records/edit_fields/attribution_partials/text_field", field_options %>
+		<%= render "records/edit_fields/attribution_partials/text_field", field_options unless template.include? "redlands" %>
```

Comment:
I feel like this is the second time that we have included a line like this, once for Pacific and once here. 

Instead of `unless template.include? "redlands"`, could you include a Flipflop to hide the isni for the tenant and that could be used on pacific as well.